components/ratgdo/ratgdo.cpp

```diff
@@ -439,22 +438,24 @@ namespace ratgdo {
 
     void RATGDOComponent::sync()
     {
-        this->rollingCodeUpdatesEnabled_ = false;
-        for (int i = 0; i <= MAX_CODES_WITHOUT_FLASH_WRITE; i++) {
-            transmit(command::GET_OPENINGS); // get openings
-            delay(65);
```

Comment:
This was in there in case the esp lost power before it wrote the counter to ensure the counter never went backwards

Reply:
I see, I'll put that back in and update the PR.